test/addons/callback-scope/test-resolve-async.js

```diff
@@ -5,9 +5,6 @@ const assert = require('assert');
 const { testResolveAsync } = require(`./build/${common.buildType}/binding`);
 
 let called = false;
-testResolveAsync().then(common.mustCall(() => {
-  called = true;
-}));
+testResolveAsync().then(() => (called = true));
 
-setTimeout(common.mustCall(() => { assert(called); }),
-           common.platformTimeout(20));
+setTimeout(() => assert(called), common.platformTimeout(50));
```

Comment:
My usual comment: I'd prefer the `assert(called)` be wrapped in curly braces to make it clear that there is no intention of returning a value here. I suspect I'm on the losing end of this particular unimportant style debate. :-P

Reply:
I don't have a strong preference so I've gone ahead and changed it.